docker/INSTALL.md

```diff
@@ -70,3 +89,20 @@ You can down the containers with
 ```
 make stop
 ```
+
+Compile webpack assets with npm and Laravel mix
+Install  `npm install`
+Compile assets `npm run prod`
+If there is some error at this step in webpack.mix.js at the end there is a timeout, add more time to it
+For example 
+```javascript
+// Ugly fix for https://github.com/StartBootstrap/startbootstrap-sb-admin-2/issues/303
+setTimeout(() => {
+    const sbAdmin2Js = `${vendorPath}/sb-admin-2/js/sb-admin-2.min.js`;
+    const content = fs.readFileSync(sbAdmin2Js, 'utf8')
+        .replace('width()<480&&', 'width()<480&&false&&');
+
+    fs.writeFileSync(sbAdmin2Js, content, 'utf8');
+}, 5000);
+```
```

Comment:
Same thing here, might be easier to directly slightly increase the delay directly in `webpack.mix.js`